dockstore-webservice/src/main/java/io/dockstore/webservice/core/User.java

```diff
@@ -80,25 +86,12 @@
     @ApiModelProperty(value = "Indicates whether this user is an admin", required = true, position = 2)
     private boolean isAdmin;
 
-    @Column
-    @ApiModelProperty(value = "Company of user", position = 3)
-    private String company;
-
-    @Column
-    @ApiModelProperty(value = "Bio of user", position = 4)
-    private String bio;
-
-    @Column
-    @ApiModelProperty(value = "Location of user", position = 5)
-    private String location;
-
-    @Column
-    @ApiModelProperty(value = "Email of user", position = 6)
-    private String email;
-
-    @Column
-    @ApiModelProperty(value = "URL of user avatar on Github.", position = 7)
-    private String avatarUrl;
+    @ElementCollection(targetClass = Profile.class)
+    @JoinTable(name = "user_profile", joinColumns = @JoinColumn(name = "id"), uniqueConstraints = @UniqueConstraint(columnNames = { "id",
+            "token_type" }))
+    @MapKeyColumn(name = "token_type", columnDefinition = "text")
+    @ApiModelProperty(value = "Profile information of the user attained from 3rd party sites (GitHub, Google, etc)")
```

Comment:
I think I prefer "retrieved" or "synced" as from the gui

attained: succeed in achieving (something that one desires and has worked for).
"clarify your objectives and ways of attaining them"